src/core/client/stream/tabs/Profile/Settings/SettingsContainer.css

```diff
@@ -0,0 +1,3 @@
+.root {
+  max-width: 440px;
```

Comment:
Ditto about internal browser text size usability with large default font sizes. How does this behave?